cmake/modules/AddSwift.cmake

```diff
@@ -235,7 +238,7 @@ function(_add_variant_link_flags)
     RESULT_VAR_NAME result)
 
   if("${LFLAGS_SDK}" STREQUAL "LINUX")
-    list(APPEND result "-lpthread" "-ldl")
+    list(APPEND result "-lpthread" "-ldl" "-lBlocksRuntime")
```

Comment:
Yes, I was wondering about this as well. Can do! I'll add these flags to `add_sourcekit_library()` instead of here.